src/EduardTrayan/WebsiteBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,7 @@
+parameters:
+    eduardtrayan.common.abstract_controller: EduardTrayan\WebsiteBundle\Controller\IndexController
```

Comment:
Please change parameter key from `abstract_controller` to `index_controller`, because you define `IndexController`